Update farmlands required fields (#391)
centrapay/centrapay-docs: src/api/batch/farmlands-external-asset.md

{% h4 Fields %}

|   Field         |        Type        |                                  Description                                    |
| :---------      | :----------------- | :------------------------------------------------------------------------------ |

> | externalId      | String {% opt %}   | Placeholder field reserved future use, expected to be unique                    |

Reviewer: ```suggestion
| externalId      | String {% opt %}   | Farmlands unique identifier for the card.                     |
```
